macos/FluentUITestApp/Assets.xcassets/Colors/DemoPrimaryShade10Color.colorset/Contents.json

```diff
@@ -0,0 +1,38 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
```

Comment:
where is this demo primary colo coming from? also, json file isn't set as hex it is floating number which won't be consistent with design figma.

Reply:
@harrieshin This is coming from the iOS demo app. I copied them from the resources folder, don't know why they got redefined in floating point. I will change them to hex.